packages/pwa-buildpack/lib/Utilities/__tests__/loadEnvironment.spec.js

```diff
@@ -87,9 +87,14 @@ test.skip('throws on load if variable defs are invalid', () => {
         ],
         changes: []
     });
-    const fn = () => loadEnvironment('./');
 
-    expect(fn).toThrow('Bad environment variable definition');
+    try {
+        await loadEnvironment('./');
+    } catch (e) {
+        expect(e.message.replace(/\n/g, '')).toBe(
+            'Bad environment variable definition. Section inscrutable variable { "type": "ineffable"} declares an unknown type ineffable'
+        );
+    }
```

Comment:
This test wouldn't fail with the expect inside the catch if this method ever stopped throwing. Async rejections are tough in jest, but I think this would work (you can swap out the inline snapshot if that's not your jam).

```suggestion
await expect(loadEnvironment('./')).rejects.toThrowErrorMatchingInlineSnapshot()
```

Reply:
Nice to know. I had no idea about `toThrowErrorMatchingInlineSnapshot`. Thanks, @tjwiebell.